modules/imports/src/main/scala/org/dhallj/imports/Canonicalization.scala

```diff
@@ -55,8 +47,10 @@ object Canonicalization {
         child match {
           //Also note that if the path is absolute then this violates referential sanity but we handle that elsewhere
           case Local(path2) =>
-            if (path2.isAbsolute) canonicalize(Classpath(path2))
-            else canonicalize(Classpath(path.getParent.resolve(path2)))
+            for {
+              parent <- LocalFile[F](path)
```

Comment:
It's not a big deal, but what do you think about using `F.map2` instead of `for`-comprehensions in cases like this where there's not a data dependency? It's a little more correct, and in my view it also reads a bit more nicely.